var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -14,14 +14,13 @@ class Relion(CMakePackage, CudaPackage):
 
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     git      = "https://github.com/3dem/relion.git"
+    url      = "https://github.com/3dem/relion/archive/3.1.0.zip"
 
-    # 3.1 is beta but referenced in published papers
-    # prefer stable 3.0 until no longer beta
-    version('3.1_beta', branch='ver3.1')
+    version('3.1.1', sha256='63e9b77e1ba9ec239375020ad6ff631424d1a5803cba5c608c09fd44d20b1618', preferred=True)
```

Comment:
No need for `preferred=True` here, Spack will already default to the newest stable release